openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -145,3 +153,75 @@ def perform_create(self, serializer):
         Create a new taxonomy.
         """
         serializer.instance = create_taxonomy(**serializer.validated_data)
+
+
+class ObjectTagView(ReadOnlyModelViewSet):
+    """
+    View to retrieve paginated ObjectTags for a provided Object ID (object_id).
```

Comment:
```suggestion
    View to retrieve paginated ObjectTags for an Object, given its Object ID.
    (What tags does this object have?)
```

This is _really_ a small nit, which I'm hesitant to even mention... but conceptually the tags aren't describing the Object ID, they're describing the object itself. So even though how you describe this now is accurate, I think it's better to phrase it along the lines I've suggested here.

Reply:
Makes sense, updated the docstrings.